src/components/app/albumsongscontainer/songinfo/song-info.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react'
+import { observer, inject } from 'mobx-react'
+import { VictoryChart, VictoryBar, VictoryTheme, VictoryPolarAxis } from 'victory'
+
+@inject('songsStore')
+@observer
+class SongInfo extends Component {
+  render() {
+    const { songsStore } = this.props
+
+    const sampleData = [
```

Comment:
Hmm wyciągnij to poza klasę, bo nie ma powodu, żeby to tutaj było + nazwa nie pasuje. To nie jest sampleData tylko bardziej labels albo coś takiego.

Reply:
Zmienilem nazwe na chartData bo to nie tlyko labelsy w sumie. Poza klase do osobnego pliku? Czy gdzie?

Reply:
W tym samym pliku, ale poza definicję klasy dla porządku :P Bo nie jest to coś co należy do klasy.

Reply:
Troche nie czaje. :p Czemu nie jest to cos, co nalezy do tej klasy, skoro uzywa injectowanego store'a z tej klasy i jest przez ta klase wykorzystywane?

Reply:
Sry źle popatrzyłem. Najważniejszym problem jest to, że to jest stała, coś co się nie zmienia, a metoda render jest wywoływana z każdym re-renderm komponentu, czyli za każdym razem tworzysz niepotrzebnie nową zmienną `sampleData`.

I chodziło mi o to, że nie używasz tego jako `this.sampleData` lub `SongInfo.sampleData` tylko `sampleData`, więc czemu miało być w definicji klasy, ale problem z re-renderem ważniejszy :P

Reply:
🤦‍♂️ My bad... Dobrze jest (w miarę :P). Nie wiem czemu ja to widziałem jako stałą...